collector/prometheus.go

```diff
@@ -38,16 +38,16 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	costmgmtv1alpha1 "github.com/project-koku/korekuta-operator-go/api/v1alpha1"
+	kokumetricscfgv1alpha1 "github.com/project-koku/koku-metrics-operator/api/v1alpha1"
 )
 
 var (
-	promSpec *costmgmtv1alpha1.PrometheusSpec
+	promSpec *kokumetricscfgv1alpha1.PrometheusSpec
 
-	costMgmtNamespace  = "openshift-cost"
-	secretKey          = "token"
-	serviceAccountName = "default"
-	tokenRegex         = "default-token-*"
+	kokuMetricsCfgNamespace = "openshift-cost"
```

Comment:
My only other comment would be we should discuss if we want to change the namespace for our community operator. I'm not sure if it matters but regardless we can do it in a different PR if we decide to!

Reply:
converted namespace to `koku-metrics-operator`